bot.js

```diff
@@ -43,9 +43,9 @@ function postCardImageTweet(status, cardName) {
 				} else {
 					logError('Unable to delete local image file: ' + err);
 				}
-			});
-		})
-		.catch(e => logError('Failed to download image: ' + e));
+      });
```

Comment:
you might want to append ?w=1 to the diff URL as there are some whitespace changes in this PR